app/src/main/java/nethical/digipaws/Constants.java

```diff
@@ -1,22 +1,22 @@
 package nethical.digipaws;
```

Comment:
why are you using 2 space tab instead of 4?
you are already using 4 space tab in some files.